yarn.lock

```diff
@@ -1620,7 +1613,7 @@ find-cache-dir@^3.2.0:
     make-dir "^3.0.0"
     pkg-dir "^4.1.0"
 
-find-java-home@^1.1.0:
+find-java-home@^1.0.1:
```

Comment:
@manisfdcsfdx  can you see is there an discrepancy here?, seeems like it got downgraded